capabilities/display-name.md

```diff
@@ -0,0 +1,19 @@
+---
+displayName: Display Name
+epicsLabels:
+  - area/displayName1
+  - area/displayName2
+---
+<!-- above metadata will be used on kyma.project.io page to display nice name of capability and have a reference to label that should be used while fetching from ZenHub/GitHub the information about related Epics and their delivery plan   -->
+
+## Goal
```

Comment:
To me the difference between "goal" and "vision" is not clear enough as both represents a future target state to achieve. 

I'd replace "goal" with "scope" or "description" to make it clear what is part of the capability and what role it plays in the context of kyma.

Reply:
O like `scope`